cps/templates/detail.html

```diff
@@ -58,6 +58,27 @@
       </div>
       </p>
       {% endif %}
+      
+      {% if cc|length > 0 %}
+        <p>
+        <div class="custom_columns">
+        {% for c in cc %}
+          {% if entry['custom_column_' ~ c.id]|length > 0 %}
+            {{ c.name }}: 
+            {% for column in entry['custom_column_' ~ c.id] %}
+              {% if c.datatype == 'rating' %}
+                {{ '%d' % (column.value / 2) }}
+              {% else %}
+                {{ '%d' % (column.value / 2) }}
```

Comment:
Typo? This are all the other columns besides rating, so just strings:
`{{ column.value }}`